.github/workflows/ci.yml

```
@@ -14,17 +14,38 @@ on:
 
 jobs:
-  test:
-    runs-on: ${{ matrix.os }}
+  unit-test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
```

actionlint (1.7.9)

the runner of "actions/setup-python@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)